API/SSW.Rewards.Infrastructure/EmailService.cs

```diff
@@ -0,0 +1,79 @@
+﻿using FluentEmail.Core;
+using Microsoft.Extensions.Logging;
+using SSW.Rewards.Application.Common.Interfaces;
+using SSW.Rewards.Application.Common.Models;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SSW.Rewards.Infrastructure
+{
+    public class EmailService : IEmailService
+    {
+        private readonly IFluentEmail _fluentEmail;
+        private readonly ILogger<EmailService> _logger;
+
+        public EmailService(IFluentEmail fluentEmail, ILogger<EmailService> logger)
+        {
+            _fluentEmail = fluentEmail;
+            _logger = logger;
+        }
+
+        public async Task<bool> SendDigitalRewardEmail(string to, string toName, string subject, DigitalRewardEmail emailProps, CancellationToken cancellationToken)
+        {
+            try
+            {
+                var template = "<p>Hi SSW Marketing</p><p>@Model.RecipientName has claimed @Model.RewardName.</p></p>Please generate a voucher code and send it to @Model.RecipientEmail.</p><p>Thanks,</p><p>SSW Rewards Notification Service</p>";
```

Comment:
Can you move the templates to a constants file?

Reply:
When we update this to have customer facing emails, they will be Razor templates stored in .cshtml files. As this is a temporary measure, I didn't see the value in storing them in separate constants files. I can do though if you prefer; alternatively I can add some comments here explaining that we will move to cshtml Razor files.